app/poros/media.rb

```diff
@@ -10,24 +10,26 @@ class Media
               :trailer,
               :language,
               :audience_score,
-              :trailer
+              :trailer, 
+              :vote_average
 
 
   def initialize(data)
     @id = data[:id]
     @title = data[:attributes][:title]
-    @poster = data[:attributes][:poster]
+    @poster = data[:attributes][:poster] || data[:attributes][:poster_path]
```

Comment:
when a poster is sent it is a full URL from wathcmode
whereas poster path is partial (only URI) from TMDB so I *think* they should be separate attributes, but open to discuss for sure.

Reply:
Made this update for clarity!